docs/data-product-studio/snowtype/snowtype-config/index.md

```diff
@@ -83,6 +83,23 @@ options?: {
 }
 ```
 
+### `namespace`
+
+:::info
+
+This option only applies when generating Swift
+
+:::
+
+The namespace for the generated code. All classes generated in the code will be included in this namespace, which can be used to avoid naming conflicts.
```

Comment:
All classes generated will be included in this namespace, which can be used to avoid naming conflicts.